CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 13.3.0 (08/01/23)
```

Comment:
Another change that I'm backporting https://github.com/gravitational/teleport/pull/29754
> Add `max_duration` role option allowing to request role access up to 7 days